src/main/java/edu/ucsb/cs156/courses/entities/Courses.java

```diff
@@ -0,0 +1,27 @@
+package edu.ucsb.cs156.courses.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "courses")
+public class Courses {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  private String enrollCd;
+  private String psld;
```

Comment:
Should this be psId (P S I D ) as in personal schedule identification) as opposed to psld ? (P S L D)

Reply:
Good catch, didn't see this one when I reviewed!